deploy/gcp/examples/tidb-customized.tfvars

```diff
@@ -1,25 +0,0 @@
-pd_instance_type   = "n1-standard-2"
```

Comment:
deprecated, so we don't encourage users to configure the tidb cluster via terraform values.